spec/controllers/admin/cms/collections_controller_spec.rb

```diff
@@ -12,6 +12,13 @@
     )
   end
 
+  before do
```

Comment:
These three are all the same, right? Is this app using Devise, if so, there are some test helper methods that they provide (described in the README) that allow you to just do something like:

```ruby
let(:user) { User.create! ...create an admin user... }
before { sign_in user }
```